Makefile

```diff
@@ -6,6 +6,8 @@ OS_ARCH = ""
 ifeq ($(OS),Windows_NT)
 	ifeq ($(PROCESSOR_ARCHITEW6432),AMD64)
 		OS_ARCH := windows_amd64
+	else ifeq ($(PROCESSOR_ARCHITEW6432),ARM64)
+		OS_ARCH := windows_arm64
```

Comment:
Not sure if this piece of code is still used. Note that `PROCESSOR_ARCHITEW6432` is only available when running a 32-bit command prompt/PowerShell on a 64-bit system (AMD64 or ARM64). If this is still used, we should probably add `ifeq ($(PROCESSOR_ARCHITECTURE),ARM64` as well, in case folks are running native ARM64 command prompt/PowerShell.